modules/post-install-extend-existing-to-zone-type.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/aws-compute-edge-tasks-zones.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="post-install-existing-local-zone-subnet_{context}"]
+= Create network requirements in an existing VPC to use an AWS zone
+
+If you want a Machine API to create an Amazon EC2 instance in a remote zone location, you must create a subnet in a {zone-type} location. You can use any provisioning tool, such as Ansible or Terraform, to create subnets in the existing Virtual Private Cloud (VPC).
+
+You can configure the CloudFormation template to meet your requirements.
+
+The following subsections include steps that use CloudFormation templates to create the network requirements that extend an existing VPC to use an AWS {zone-type}.
+
+For AWS Local Zones, consider the limitation of NAT Gateways in that CloudFormation templates support only public subnets. You can reuse the template to create public subnets for each edge location to where you need to extend your cluster.
```

Comment:
There is no more limitation of public subnet, the new template for subnet cover both private and public.

```suggestion
For AWS Local Zones and Wavelength Zones, consider the limitation of NAT Gateways in that CloudFormation templates supports only associating private subnets with private route tables in the region.

For Local Zones, on completing these steps in the following subsections, you create the following resources:

* 2 VPC Subnets: public and private. The public subnet is associated to the public route table for the regular Availability Zones in the region. The private subnet is associated to the provided route table ID.

```

Reply:
Updated. Should "region" be "Region"?